kompari-tasks/src/args.rs

```diff
@@ -49,4 +49,16 @@ pub struct SizeCheckArgs {
     /// If enabled, images on file system are replaced with optimized version
     #[arg(long, default_value_t = false)]
     pub optimize: bool,
+
+    /// Command will fail if at least one image can be optimized by more than given ratio.
+    /// E.g. --improvement-limit=0.8 means that error is signaled when an image can be optimized
+    /// more than 80% of its original size
```

Comment:
```suggestion
    /// E.g. --improvement-limit=0.8 means that error is signaled when an image can be optimized
    /// more than 80% of its original size (that is, the optimised image's size is 20% or less of the original)
```

This is the kind of thing which is really hard to say unambiguously, because this could read either as "the optimised image is less than 20% of the original size, or the optimised image is less than 80% of the original size".